releasenotes/notes/new-app-kubernetes-name.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+issue: [52034]
+releaseNotes:
+- |
+  **Added** Ingress and Egress gateways now have the `app.kubernetes.io/name=istiod` label.
```

Comment:
This is the webhooks not gateways?

I think we don't need a release note though, since this is just tweaking a PR that was merged but hasn't yet shipped to users

Reply:
Oh fair enough - I missed that this is just the new label, and thought this was something we were already doing.